code/modules/mob/living/simple_animal/abnormality/zayin/fallen_amurdad.dm

```diff
@@ -244,3 +299,128 @@
+		if(AV in protected_plants)
+			AV.obj_integrity += 50
+			continue
+		protected_plants += AV
+		AV.max_integrity = 300
+		AV.obj_integrity = 300
+
+/obj/structure/amurdad_bomb/proc/Grow()
+	stage = stage + 1 > 5 ? 5 : stage + 1
+	UpdateStage()
+	ProtectPlants()
+	if(stage >= 5)
+		return
+	addtimer(CALLBACK(src, .proc/Grow), grow_interval)
+
+/obj/structure/amurdad_bomb/proc/UpdateStage()
+	cut_overlays()
+	if(stage == 0)
+		return
+	var/mutable_appearance/plant_overlay = mutable_appearance('icons/obj/hydroponics/growing.dmi', "deathnettle-grow[stage]", layer = OBJ_LAYER + 0.01)
+	add_overlay(plant_overlay)
+
+/obj/structure/amurdad_bomb/HasProximity(atom/movable/AM)
+	if(stage <= 4)
+		return
+	if(!isliving(AM))
+		return
+	var/mob/living/L = AM
+	if(("hostile" in L.faction))
+		return
+	INVOKE_ASYNC(src, .proc/Explode)
+
+/obj/structure/amurdad_bomb/bullet_act(obj/projectile/P)
+	. = ..()
+	if(stage <= 3)
+		return
+	Explode()
+
+/obj/structure/amurdad_bomb/proc/Explode()
+	det_com = AddComponent(/datum/component/pellet_cloud, projectiletype, stage)
+	SEND_SIGNAL(src, COMSIG_MINE_TRIGGERED)
+	stage = 0
+	UpdateStage()
+	addtimer(CALLBACK(src, .proc/Grow), grow_interval)
+
+/obj/projectile/needle
+	name = "venomous needle"
+	desc = "a venomous thorn from a plant"
+	icon_state = "needle"
+	ricochet_chance = 60
+	ricochets_max = 2
+	damage = 3
+	damage_type = RED_DAMAGE
+	eyeblur = 2
+	ricochet_ignore_flag = TRUE
+
+/obj/projectile/needle/can_hit_target(atom/target, direct_target, ignore_loc, cross_failed)
+	if(!fired)
+		return FALSE
+	return ..()
+
+/obj/projectile/needle/on_hit(atom/target, blocked, pierce_hit)
+	. = ..()
+	if(ishuman(target))
+		var/mob/living/carbon/human/H = target
+		H.reagents.add_reagent(/datum/reagent/toxin/amurdad_poison, 10)
+
+/datum/reagent/toxin/amurdad_poison
+	name = "Fallen Nightshade"
+	description = "A poison that corrupts the blood and turns the stomach foul."
+	taste_description = "iron"
+	glass_name = "glass of fallen nightshade"
+	glass_desc = "It smells of roses and yet looks awful."
+	color = "#433748"
+	can_synth = FALSE
+	harmful = TRUE
+	toxpwr = 0
+	metabolization_rate = REAGENTS_METABOLISM * 4
+
+/datum/reagent/toxin/amurdad_poison/on_mob_metabolize(mob/living/L)
+	. = ..()
+	to_chat(L, "<span class='warning'>You feel nauseous...</span>")
+
+/datum/reagent/toxin/amurdad_poison/on_mob_end_metabolize(mob/living/L)
+	. = ..()
+	to_chat(L, "<span class='nicegreen'>You start to feel better.</span>")
+
+/datum/reagent/toxin/amurdad_poison/on_mob_life(mob/living/M)
+	var/damage_mod = 1
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		var/obj/item/organ/liver/LV = locate() in H.internal_organs
+		if(volume <= LV.toxTolerance)
+			return ..()
+		damage_mod = LV.toxLethality * 100
+	metabolization_rate = max(volume * REAGENTS_METABOLISM, REAGENTS_METABOLISM)
+	M.apply_damage(volume*REAGENTS_METABOLISM*damage_mod, RED_DAMAGE, null, M.run_armor_check(null, BLACK_DAMAGE), spread_damage = TRUE)
```

Comment:
It deals red damage but checks for black.

Reply:
This was intentional at the time, can and likely will be changed